### PR TITLE
CATTY-468: Create Stitch Brick

### PR DESCRIPTION
**Pull request by Bomml**
I created a new Brick named StitchBrick which is the first step for the Embroidery feature. A stitch should be a small dots on the Stage and two stitches should be connected with a line:

I created a small TestProgram. You can download it:
https://share.catrob.at/app/project/b365cb6d-327d-11eb-a92d-005056a36f47

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catty/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Verify that the Jira ticket is in the status *Ready for Development*
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#catty* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by aeluna — APPROVED** [review]
Thank you for the PR!

File: src/Catty/Defines/LanguageTranslationDefines.h [hunk 1]
```diff
@@ -247,6 +247,7 @@
 #define kLocalizedLists NSLocalizedString(@"Lists", nil)
 #define kLocalizedPhiroBricks NSLocalizedString(@"Use Phiro bricks", nil)
 #define kLocalizedArduinoBricks NSLocalizedString(@"Arduino extension", nil)
+#define kLocalizedEmbroideryBricks NSLocalizedString(@"Use embroidery bricks", nil)
```
Comment (m-herold) [inline src/Catty/Defines/LanguageTranslationDefines.h:250]: Please rename to "Embroidery extension". And add the following subtitle/description to the settings page:

> Create patterns for stitching machines

File: src/Catty/Setup/CatrobatSetup+Bricks.swift [hunk 2]
```diff
@@ -178,6 +180,12 @@
                                             color: UIColor.arduinoBrick,
                                             strokeColor: UIColor.arduinoBrickStroke))
         }
+        if isEmbroideryEnabled() {
```
Comment (m-herold) [inline src/Catty/Setup/CatrobatSetup+Bricks.swift:183]: If the embroidery extension is enabled, please add as first category (above "Event").

**Review by m-herold — CHANGES_REQUESTED** [review]
Thanks for this great PR. I have left a few suggestions.

**Review by m-herold — APPROVED** [review]
Thanks!

**Pull request merged**